torchdata/nodes/samplers/multi_node_weighted_sampler.py

```diff
@@ -201,6 +201,22 @@ def get_state(self) -> Dict[str, Any]:
 
 
 class _WeightedSampler:
+    """A weighted sampler that samples from a list of weights.
```

Comment:
nit: this is private so we it's not strictly necessary/lower priority to document it